### PR TITLE
Add: Cancel glitch user actions

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ClobberXD** [comment]
`setpos` and `getpos` are deprecated. Use `set_pos` and `get_pos` instead.

File: mods/ctf_pvp_engine/ctf_flag/flag_func.lua [hunk 1]
```diff
@@ -210,6 +210,14 @@ ctf_flag = {
 					minetest.chat_send_player(name, "You are at peace with this team!")
 					return
 				end
+				
+				local glitcher = players_glitching[name]
+				if glitcher then
+					local player = minetest.get_player_by_name(name)
+					player:setpos(glitcher)
+					minetest.chat_send_player(name, "You can not capture a flag while in airplane mode!")
```
Comment (ClobberXD) [inline mods/ctf_pvp_engine/ctf_flag/flag_func.lua]: Could do with a more generic message like: `"You can't capture the flag by glitching!"`

File: mods/ctf_pvp_engine/ctf_flag/flag_func.lua [hunk 2]
```diff
@@ -210,6 +210,14 @@ ctf_flag = {
 					minetest.chat_send_player(name, "You are at peace with this team!")
 					return
 				end
+				
+				local glitcher = players_glitching[name]
```
Comment (ClobberXD) [inline mods/ctf_pvp_engine/ctf_flag/flag_func.lua]: What's actually stored in `players_glitching[name]` is a pos. You might want to rename `glitcher` to `pos` or something to eliminate confusion.

File: mods/ctf_pvp_engine/ctf_flag/flag_func.lua [hunk 3]
```diff
@@ -210,6 +210,14 @@ ctf_flag = {
...
+				if glitcher then
+					local player = minetest.get_player_by_name(name)
```
Comment (ClobberXD) [inline mods/ctf_pvp_engine/ctf_flag/flag_func.lua]: `player` is only used in the next line after which it's scope expires. You can directly call `set_pos` like so:

`minetest.get_player_by_name(name):set_pos(pos)`

**Review by ClobberXD — COMMENTED** [review]
LGTM code-wise, but I hope this has been tested thoroughly to ensure that it works correctly.

**Comment by ghost** [comment]
The code works in singleplayer but I am un-able to test it without a server.

**Comment by LoneWolfHT** [comment]
> The code works in singleplayer but I am un-able to test it without a server.

Host a world. Then open another minetest window and join `127.0.0.1:30000`

**Comment by ghost** [comment]
But how can I disconnect and reconnect in singleplayer? thats the part that needs testing.

**Comment by LoneWolfHT** [comment]
> But how can I disconnect and reconnect in singleplayer? thats the part that needs testing.

Ah, ok.

**Comment by ClobberXD** [comment]
Host a server on a PC, and connect with your phone as the client.

**Pull request merged**